test-cypress/integration/home/workbench.home.spec.js

```diff
@@ -220,4 +266,17 @@ describe('Home screen validation', () => {
         });
     });
 
+    function getRDFResourceSearchBox() {
```

Comment:
You should check if the button is visible before interacting with it. Should will try to reload the button until condition is found.